translations/localify/ui.json

```diff
@@ -1997,5 +1998,158 @@
     "このレースで{0}着以内になる": "Place top {0} in this race",
     "現在選択しているウマ娘を編成すると、\n上記のウマ娘がチームから外れますがよろしいですか？": "Adding these horsegirls will remove the following \nones from the team, are you sure?",
     "出走履歴がありません": "Race history lacking",
-    "編成時に対象キャラをハイライトする": "Highlight these chars during set up"
+    "編成時に対象キャラをハイライトする": "Highlight these chars during set up",
+    "<chrname>は、\nなにか得るものがあったようだ！": "<chrname> seems to have gained something! ",
+    "その時、ふと閃いた！\nこのアイディアは、<chrname>との\nトレーニングに活かせるかもしれない！": "Just then, I had an idea! \nThis might help <chrname>'s training!",
+    "<chrname>の成長につながった！": "It helped <chrname> grow! ",
+    "お休み": "Rest",
+    "1月": "Jan",
+    "2月": "Feb",
+    "3月": "Mar",
+    "4月": "Apr",
+    "5月": "May",
+    "6月": "Jun",
+    "7月": "Jul",
+    "8月": "Aug",
+    "9月": "Sep",
+    "10月": "Oct",
+    "11月": "Nov",
+    "12月": "Dec",
+    "育成データ削除の確認": "Confirm training data deletion",
+    "育成データ削除を確認したため\nタイトルに戻ります": "Due to confirmation of training data deletion \nreturning to title",
+    "{0}億人": "{0}00,000,000 Fans",
+    "{0}万人": "{0}0,000 Fans",
+    "+{0}人": "+{0} Fans",
+    "+{0}万人": "+{0}0,000 Fans",
+    "おすすめトレーナーがいません": "No recommended trainers",
+    "1分以内": "In 1 min",
```

Comment:
Can you confirm where these are used?
It sounds more like "within" or "last" x min like would appear in a message log (circle chat probably).